views/editor/publish.html

```diff
@@ -16,10 +16,17 @@
     </div>
 
     <div id="publish-live" class="hide">
+     
       <p>{{ gettext("publishShareLink") }}</p>
       <div id="publish-link">
         <a id="link-publish-link" title="{{ gettext("publishShareLinkTitle") }}" target="_blank" href="test"></a>
       </div>
+      
+        <!--   Embed iFrame   -->
+      <p>{{ gettext("publishEmbedLinkTitle") }}</p>
+      <div id="publish-link">
+          <span>&lt;iframe {{ gettext("PUBLISHED_PROJECTS_HOSTNAME") }} &gt; </span>
```

Comment:
I'm not sure why "PUBLISHED_PROJECTS_HOSTNAME" would be used here.
Instead, leave the text empty and dynamically change the text with JS instead:
1. Add a property [here](https://github.com/mozilla/thimble.mozilla.org/blob/master/public/editor/scripts/ui/publisher.js#L37) called `embed` that is a jquery object linking to this span.
2. After [this line](https://github.com/mozilla/thimble.mozilla.org/blob/master/public/editor/scripts/ui/publisher.js#L315) set the text of `embed` to be the string ``` `<iframe src={$publishUrl}></iframe>` ```